internal/status/ing_status.go

```diff
@@ -261,15 +260,25 @@ func isAnnotationsUpdateRequired(ingAnnotations map[string]string, newVSAnnotati
 			return true
 		}
 	}
+	if !isDelete {
+		oldTenant, ok := ingAnnotations[lib.TenantAnnotation]
```

Comment:
I think we can add the tenant info to VS annotation itself instead of sending as a separate field. And we can handle the case of VsDelete by just skipping the tenant addition to vs annotation(assuming it is in a single place)

Reply:
It seems much cleaner to add a new field since vsuuid Annotations is already a host-vsuuid map